app/views/assignments/view_summary.js.erb

```diff
@@ -1,14 +1,14 @@
 document.getElementById('assignment_summaries').innerHTML =
   "<%= escape_javascript(render partial: 'assignment_summary',
-                                locals: { assignment: @assignment, ta: @current_ta}).html_safe %>";
+                                locals: { assignment: @assignment, ta: @tas}).html_safe %>";
```

Comment:
Change the local variable here to `tas` (from the singular `ta`) to better reflect the relevant value. Note that you need to change this in a couple of places (in all the relevant files).